modules/e2ee/E2EEncryption.js

```diff
@@ -89,7 +89,7 @@ export class E2EEncryption {
      * @param {Participant} - participant to be verified.
      * @returns {void}
      */
-    markChannelVerified(isVerified, participant) {
-        this._keyHandler.sasVerification?.markChannelVerified(isVerified, participant);
+    markParticipantVerified(isVerified, participant) {
```

Comment:
Please reverse this parameters, we usually put them that way, like when sending messages.